### PR TITLE
feat: pnp

### PR DESCRIPTION
**Pull request by kpanot**

### CODE REVIEW

**Review by matthieu-crouzet — DISMISSED** [review]

**Review by matthieu-crouzet — DISMISSED** [review]

File: packages/@ama-sdk/core/package.json [hunk 1]
```diff
@@ -43,19 +42,14 @@
...
-  "peerDependenciesMeta": {
-    "@angular-eslint/template-parser": {
-      "optional": true
-    },
-    "@angular/compiler": {
-      "optional": true
-    }
-  },
   "peerDependencies": {
     "isomorphic-fetch": "^2.2.1"
```
Comment (vscaiceanu-1a) [inline packages/@ama-sdk/core/package.json:46]: shouldn't `"@angular-devkit/schematics": "~15.0.4",` be part of the peerDeps as well? isn't `yarn in-source-dep-check` flagging this as a warning?

**Review by vscaiceanu-1a — COMMENTED** [review]
if you run `npm_config_yes=true npx yarn-audit-fix@latest` (equivalent of `npm audit --fix`) it will bump some patches and fix some dependabot alerts. This way we can discard https://github.com/AmadeusITGroup/otter/pull/21/files

File: packages/@o3r/storybook/package.json [hunk 2]
```diff
@@ -59,13 +59,20 @@
     "@storybook/angular": "^6.5.14",
     "@storybook/api": "^6.5.14",
     "@storybook/components": "^6.5.14",
+    "comment-json": "^4.1.0",
```
[the remark this answers is not in the document]
Comment (matthieu-crouzet) [reply packages/@o3r/storybook/package.json:62]: This package is using @o3r/schematics which has comment-json in peerDep
so it's seems logic to have it as peerDep optional here

File: packages/@ama-sdk/core/package.json (hunk 1 above)
Comment (matthieu-crouzet) [reply packages/@ama-sdk/core/package.json:46]: @angular-devkit/schematics is use only for an import type so devDep is enough I think

File: packages/@ama-sdk/core/package.json [hunk 3]
```diff
@@ -43,19 +42,14 @@
     "tslib": "^2.4.1",
     "uuid": "^8.3.2"
   },
-  "peerDependenciesMeta": {
-    "@angular-eslint/template-parser": {
```
[the remark this answers is not in the document]
Comment (matthieu-crouzet) [reply packages/@ama-sdk/core/package.json:47]: Why isomorphic-fetch is in peerDep ? It's used only for setup testing.
Shouldn't we keep it only as devDep ?

**Comment by matthieu-crouzet** [comment]
> if you run `npm_config_yes=true npx yarn-audit-fix@latest` (equivalent of `npm audit --fix`) it will bump some patches and fix some dependabot alerts. This way we can discard https://github.com/AmadeusITGroup/otter/pull/21/files

I would prefer to keep your PR apart to fix dependabot issue

**Pull request merged**